hospital/hospitalcore/resources/hospitalcore-items.xml

```diff
@@ -307,6 +307,10 @@
 					<attribute qualifier="number" type="java.lang.Integer">
 						<persistence type="property" />
 					</attribute>
+					<attribute qualifier="free" type="java.lang.Boolean">
```

Comment:
per convenzione gli attributi booleani iniziano con "is" in questo caso è preferibile "isFree"